deps/spidershim/src/v8function.cc

```diff
@@ -27,6 +27,32 @@
 
 namespace v8 {
 
+MaybeLocal<Object>
+Function::NewInstance(Local<Context> context,
+                      int argc, Handle<Value> argv[]) const
+{
+  Isolate* isolate = Isolate::GetCurrent();
+  JSContext* cx = JSContextFromIsolate(isolate);
+  JS::RootedObject thisObj(cx, &reinterpret_cast<const JS::Value*>(this)->toObject());
```

Comment:
@ehsan Is it also unnecessary to root this here, because the internal::Local<Object>::New() constructor will do that?